app/views/releases/show.html.erb

```diff
@@ -16,6 +16,13 @@
   <label>Created</label>
   <%= relative_time(@release.created_at) %>
 
+  <legend>Github Status</legend>
```

Comment:
maybe name this commit-status so it's not github specific and can serve multiple backends eventually ...